concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -5,35 +5,28 @@
 import random
 import time
 from base64                 import b64encode
+from collections import namedtuple
 from freezegun              import freeze_time
 
 from golem_messages         import message
 
 from utils.helpers          import get_current_utc_timestamp
 from utils.testing_helpers  import generate_ecc_key_pair
 
-from api_testing_helpers    import api_request
-from api_testing_helpers    import timestamp_to_isoformat
+from api_testing_common import api_request, parse_command_line
+from api_testing_common    import timestamp_to_isoformat
 
 import requests
 
+HOUR = 3600
+TIME_GUARD = 10
```

Comment:
A comment explaining this would be nice because the name on its own does not say much.

Reply:
@cameel - it's a time offset added to calculated times, to ensure tests will not fail because of 1 sec time difference on host, etc.

Reply:
I know, you explained it to me earlier. My point was that you should put this explanation in a comment in the code.